internal/ui/main_page.go

```diff
@@ -0,0 +1,53 @@
+package ui
+
+import (
+	"github.com/rivo/tview"
+)
+
+func MainPage(app *tview.Application) error {
```

Comment:
maybe wrapping `tview.Application` in another struct is a better approach e.g. [k9s](https://github.com/derailed/k9s/blob/a8e0942d7eca4fc7c9dbff698d3b0a4e5a7bf235/internal/ui/app.go#L16) or [mangadesk](https://github.com/darylhjd/mangadesk/blob/20e3d4206bae4fac79d1f85c8a7d78346fee3a6d/app/core/app.go#L19), there should be more examples out there.

Reply:
Noted, will be updated

Reply:
updated